benchmarks/run_httest_torchrun.sh

```
@@ -0,0 +1,51 @@
+if [ "${SLURM_LOCALID:-0}" = "0" ]; then
+  mkdir -p "$JIT_INC"
+  ln -sfn "$WHEEL_EP"/include/* "$JIT_INC"/ 2>/dev/null
+  ln -sfn "$NCCL_INC"/* "$JIT_INC"/ 2>/dev/null
+  touch "$JIT_INC/.ready"
+else
+  for _ in $(seq 1 50); do [ -f "$JIT_INC/.ready" ] && break; sleep 0.2; done
+fi
+export NCCL_EP_JIT_SOURCE_DIR="$WHEEL_EP/include/nccl_ep"
+export NCCL_EP_JIT_BUILD_INCLUDE_DIR="$JIT_INC"
+export NCCL_EP_JIT_CUDA_INCLUDE_DIR="${CUDA_HOME:-/usr/local/cuda}/include"
+export NCCL_EP_JIT_NVCC="$(command -v nvcc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)